db/dcrpg/pgblockchain.go

```diff
@@ -193,6 +194,21 @@ type cacheLocks struct {
 	utxo       *cache.CacheLock
 }
 
+// BlockGetter implements a few basic blockchain data retrieval functions. It is
+// like rpcutils.BlockFetcher except that it must also implement
+// GetBlockChainInfo.
+type BlockGetter interface {
+	// rpcutils.BlockFetcher implements GetBestBlock, GetBlock, GetBlockHash,
+	// and GetBlockHeaderVerbose.
+	rpcutils.BlockFetcher
+
+	// GetBlockChainInfo is required for a legacy upgrade involving agendas.
+	GetBlockChainInfo() (*dcrjson.GetBlockChainInfoResult, error)
+}
+
+// Ensure that blockGetter satisfies rpcutils.BlockFetcher.
+var _ rpcutils.BlockFetcher = BlockGetter(nil)
```

Comment:
Good point.  I still liked it there so that it makes it less likely that someone will change `BlockGetter` to not embed `rpcutils.BlockFetcher`.  However, if that were changed, it would break everywhere the `BlockGetter` was passed to `rpcutils` functions.  Will remove.